api/src/main/java/org/apache/iceberg/metrics/ScanReport.java

```diff
@@ -19,28 +19,32 @@
 package org.apache.iceberg.metrics;
 
 import java.io.Serializable;
+import java.time.Duration;
 import java.util.concurrent.TimeUnit;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.expressions.Expression;
 import org.apache.iceberg.metrics.MetricsContext.Counter;
+import org.apache.iceberg.metrics.MetricsContext.Unit;
 import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
+import org.apache.iceberg.relocated.com.google.common.base.Objects;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 
 /** A Table Scan report that contains all relevant information from a Table Scan. */
 public class ScanReport implements Serializable {
 
   private final String tableName;
   private final long snapshotId;
-  private final Expression filter;
+  // TODO: change this to Expression once we have an ExpressionParser
+  private final String filter;
```

Comment:
Can we omit this for now? If anything happens and we forget about it, I'd rather not make a breaking format change.

Reply:
ok I removed this for now as we can easily add it back later